agent/handlers/v4/response.go

```diff
@@ -104,6 +104,8 @@ func NewTaskResponse(
 		})
 	}
 
+	v2Resp.LaunchType = "EC2"
```

Comment:
Wonder if we could use the constant defined here: https://github.com/aws/amazon-ecs-agent/blob/master/agent/ecs_client/model/ecs/api.go#L12642

Reply:
good catch, I will substitute the constant

Reply:
fixed